packages/runtime-core/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './backends/local-storage.js'
+export * from './useCore.js'
```

Comment:
Maybe we can remove these extensions as we have `Bundler` as `moduleResolution`:

```suggestion
export * from './backends/local-storage'
export * from './useCore'
```